### ISSUE TITLE
Shift and unshift method names are interchanged.

### ISSUE THREAD

**Issue by vitasimek** [rlogiacco/CircularBuffer#4]
Hi,
shift and unshift method names are interchanged.
There in the doc is: "...shift() adds to the head...", but there in the code is defined as follows: "T shift();" and is commented as follows: "Removes an element from the beginning of the buffer.". Conversely the same applies for unshift method.

**Comment by rlogiacco** [comment]
Good catch, it's being fixed right now: please keep an eye on the next commit, if it's all fine please close this issue.

Thanks.
